docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -32,6 +32,21 @@ _Returns_
 
 -   `boolean`: Whether the last change was automatic.
 
+<a name="getAccessibleBlockLabel" href="#getAccessibleBlockLabel">#</a> **getAccessibleBlockLabel**
+
+Get a label for the block for use by screenreaders, this can include the block title, the
+position of the block, and the value of the `getAccessibilityLabel` function if it's specified.
+
+_Parameters_
+
+-   _state_ `Object`: Store state.
+-   _clientId_ `string`: ClientId for the block.
+-   _direction_ `[(|)]`: The direction blocks are layed out in, can be 'horizontal' or 'vertical'.
```

Comment:
This is incidental of the implementation of the direction. Ideally it's something declarative where the "clientId" is sufficient. (I believe it's available in the store already now with a recent fix by Ella)

Reply:
Oh, didn't realise that was now available in the store. Thanks for the pointer, now updated to use the store.